en/includes/deploy/deploy-is-in-kubernetes.md

```diff
@@ -0,0 +1,165 @@
+# Deploy {{product_name}} in Kubernetes using Helm
+
+This guide walks you through deploying WSO2 Identity Server as a containerized application on a Kubernetes cluster using the official Helm chart. Helm simplifies the deployment process by automating the configuration and management of Kubernetes resources, making it easier to set up and maintain your {{product_name}} instance.
+
+## Prerequisites
+
+Make sure you have the following before starting this guide.
+
+- Ensure your system has the following tools installed.
+    - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git){: target="_blank"}
+    - [Helm](https://helm.sh/docs/intro/install/){: target="_blank"}
+    - [Docker](https://docs.docker.com/engine/install/){: target="_blank"}
+    - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl){: target="_blank"}
+
+- A running Kubernetes cluster (e.g. [minikube](https://kubernetes.io/docs/tasks/tools/#minikube){:target="_blank"} or an existing cluster).
+
+- A Kubernetes [Ingress-Nginx Controller](https://kubernetes.github.io/ingress-nginx/deploy/){:target="_blank"}.
+
+## Add the WSO2 Helm chart repository
+
+Before installing {{product_name}}, add the WSO2 Helm chart repository and update it to fetch the latest charts.
+
+```shell
+helm repo add wso2 https://helm.wso2.com && helm repo update
+```
+
+## Set up environment variables
+
+Define environment variables for the Kubernetes namespace and Helm release name.
+
+```shell
+export NAMESPACE=<Kubernetes Namespace to deploy the resources>
+export RELEASE_NAME=<Helm release name for the deployment>
+```
+!!! note
+
+    - Replace <Kubernetes Namespace to deploy the resources> with the namespace where WSO2 Identity Server should be deployed.
+    - Replace <Helm release name for the deployment> with a unique name for the Helm release.
+
+## Create a Kubernetes namespace
+
+Ensure that the specified namespace exists or create a new one using the following command.
+
+```shell
+kubectl get namespace ${NAMESPACE} || kubectl create namespace ${NAMESPACE}
+```
+
+## Install the Helm chart
+
+There are two ways to install the {{product_name}} Helm chart. The Helm chart source code can be found in the [kubernets-is repository](https://github.com/wso2/kubernetes-is/tree/master){:target=" _blank"}.
+
+### Option 1: Install the chart from the Helm repository
+
+Use the following command to install {{product_name}} from the official WSO2 Helm repository:
+
+```shell
+helm install "$RELEASE_NAME" wso2/identity-server --version {{is_version}}-1 \
```

Comment:
This needs to be updated in 7.0.0 once a new chart is released

In 7.1.0, it should be kept as it is

Reply:
Give the version as <version>

Then add a note afterwards, saying to check the latest version from artifact hub
https://artifacthub.io/packages/helm/wso2/identity-server